extensions/ql-vscode/src/packages/commands/CommandManager.ts

```diff
@@ -1,9 +1,25 @@
+/**
+ * Contains a generic implementation of typed commands.
+ *
+ * This allows different parts of the extension to register commands with a certain type,
+ * and then allow other parts to call those commands in a well-typed manner.
+ */
+
 export interface Disposable {
   dispose(): void;
 }
 
+/**
+ * A command function is a completely untyped command.
+ */
 export type CommandFunction = (...args: any[]) => Promise<unknown>;
 
+/**
+ * The command manager basically takes a single input, the type
+ * of all the known commands. The second parameter is provided by
+ * default (and should not be needed by the caller) it is a
+ * technicality to allow the type system to look up commands.
```

Comment:
Thanks for clarifying 👍

Reply:
I think it's better to leave it as-is. It's not unused, it's just an optional parameter. In function arguments, we would also only add underscores if the argument is unused, rather than optional.